src/applications/financial-status-report/components/householdIncome/SpouseTransitionExplainer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ExplainerComponent from '../shared/ExplainerComponent';
+
+const SpouseTransitionExplainer = ({
+  contentBeforeButtons,
+  contentAfterButtons,
+  goBack,
+  goForward,
+}) => {
+  return (
+    <ExplainerComponent
+      headline="You added a spouse"
+      strongMessage="You will now be asked additional questions about your spouse’s
+      income and employment."
```

Comment:
Add these sentences here: 

`Your spouse is considered a dependent. On the dependents page, include your spouse as a dependent.`

Reply:
Done!